build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -93,12 +93,23 @@ spec:
           {{- else }}
           command: ["install_cni"]
           {{- end }}
+          {{- with .Values.agent.installCNI.securityContext }}
           securityContext:
+            {{- if .privileged }}
+            privileged: true
+            {{- else }}
+            {{- with .capabilities }}
             capabilities:
               add:
-                # SYS_MODULE is required to load the OVS kernel module.
-                - SYS_MODULE
+                {{- toYaml . | nindent 16 }}
+            {{- end }}
+            {{- end }}
+          {{- end }}
           env:
+            {{- range $k, $v := .Values.agent.installCNI.extraEnv }}
+            - name: {{ $k | quote }}
+              value: {{ $v | quote }}
+            {{- end }}
```

Comment:
what env vars will it be used to inject? For skipping loading OVS module?

Reply:
yes, I was thinking that it was a possible option. But in the context of this PR, I just wanted to address the discrepancy with the antrea-agent container.